package.json

```diff
@@ -195,8 +195,9 @@
     "lint:js": "eslint --ext .js,.cjs,.md .",
     "lint:js:fix": "npm run lint:js -- --fix",
     "lint:js:report": "eslint --ext .js,.cjs,.md --output-file build/lint-js-report.json --format json .",
-    "lint:php": "vendor/bin/phpcs",
+    "lint:php": "composer phpcs",
     "lint:php:fix": "vendor/bin/phpcbf",
+    "lint:phpstan": "composer phpstan",
```

Comment:
```suggestion
    "lint:php:stan": "composer phpstan",
```
Should this be?

Reply:
What'd be the benefit? Seems unnecessary.

Reply:
Just feel cleaner. It is just a suggestion.

Reply:
Leaving as-is for now